examples/stub.php

```diff
@@ -452,7 +452,7 @@ public function __construct(int $size, string $class = Worker::class, array $cto
      *
      * @link http://www.php.net/manual/en/pool.collect.php
      */
-    public function collect(callable $collector) {}
+    public function collect(callable $collector = null) {}
```

Comment:
The default is technically the built-in `Worker::collector` method (same with the `Worker::collect` stub). I'm not sure how it can really be represented in the method signature, though. `[Worker::class, 'collector']` isn't accurate, since it's not a statically callable method, and I'm not sure `[$this, 'collector']` is much better either. Thoughts? Otherwise, perhaps it should just be mentioned in the doc comment instead.

Reply:
I wasn't sure on this either. A doc comment is probably the best option.